src/hotspot/share/opto/library_call.cpp

```diff
@@ -2194,31 +2194,32 @@ bool LibraryCallKit::inline_number_methods(vmIntrinsics::ID id) {
 }
 
 //--------------------------inline_unsigned_divmod_methods-----------------------------
-// inline int Integer.divideUnsigned(init, int)
+// inline int Integer.divideUnsigned(int, int)
 // inline int Integer.remainderUnsigned(int, int)
+// inline long Long.divideUnsigned(long, long)
+// inline long Long.remainderUnsigned(long, long)
 bool LibraryCallKit::inline_divmod_methods(vmIntrinsics::ID id) {
   Node* n = NULL;
   switch(id) {
     case vmIntrinsics::_divideUnsigned_i:
       zero_check_int(argument(1));
-      // Compile-time detect of null-exception?
-      if (stopped()) return false;
-      n = new UDivINode(control(), argument(0), argument(1));
-      break;
+      // Compile-time detect of null-exception
+      if (stopped()) return true; // keep the graph constructed so far
+      n = new UDivINode(control(), argument(0), argument(1)); break;
```

Comment:
```suggestion
      if (stopped()) {
        return true; // keep the graph constructed so far
      }
      n = new UDivINode(control(), argument(0), argument(1));
      break;
```

While you are modifying this code, please also fix the code style.

Reply:
Please see the updated code style incorporating the suggested changes in the newer commit.